src/main/java/net/schmizz/sshj/transport/cipher/BaseCipher.java

```diff
@@ -38,14 +38,20 @@
     }
 
     private final int ivsize;
+    private final int authSize;
```

Comment:
Let's keep this field in the `GcmCipher` which is the only one using it as far as I'm aware, and return `0` as default return value from the `getAuthenticationTagSize()` method.